ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/PolicyViolationsBySeverity.js

```diff
@@ -10,18 +10,23 @@ import { Link, withRouter } from 'react-router-dom';
 import { gql } from '@apollo/client';
 import max from 'lodash/max';
 import { severityValues, severities } from 'constants/severities';
-import {
-    severityColorMap,
-    severityTextColorMap,
-    severityColorLegend,
-} from 'constants/severityColors';
+import { policySeverityColorMap } from 'constants/visuals/colors';
+import { severityLabels as policySeverityLabels } from 'messages/common';
 import policyStatus from 'constants/policyStatus';
 import entityTypes from 'constants/entityTypes';
 import searchContext from 'Containers/searchContext';
 import { CLIENT_SIDE_SEARCH_OPTIONS as SEARCH_OPTIONS } from 'constants/searchOptions';
 import { getPercentage } from 'utils/mathUtils';
 
-const passingLinkColor = 'var(--base-500)';
+const policySeverities = ['LOW_SEVERITY', 'MEDIUM_SEVERITY', 'HIGH_SEVERITY', 'CRITICAL_SEVERITY'];
+
+const legendData = policySeverities.map((severity) => ({
+    title: policySeverityLabels[severity],
+    color: policySeverityColorMap[severity],
+}));
+
+const linkColor = 'var(--primary-600)'; // TODO might be able to remove after PatternFly dark them
```

Comment:
nit: spelling of "theme"
```suggestion
const linkColor = 'var(--primary-600)'; // TODO might be able to remove after PatternFly dark theme
```

Reply:
Good eyes! Unintentional “othering” of theme.